drawio/docker-compose.yaml

```diff
@@ -63,7 +67,7 @@ services:
       - DRAWIO_GOOGLE_VIEWER_CLIENT_SECRET=${DRAWIO_GOOGLE_VIEWER_CLIENT_SECRET}
       - DRAWIO_MSGRAPH_CLIENT_ID=${DRAWIO_MSGRAPH_CLIENT_ID}
       - DRAWIO_MSGRAPH_CLIENT_SECRET=${DRAWIO_MSGRAPH_CLIENT_SECRET}
-      - DRAWIO_GITLAB_ID=${DRAWIO_GITLAB_ID}
+      - DRAWIO_GITLAB_ID=${DRAWIO_GITLAB_ID}ww
```

Comment:
Walter White?

Reply:
Uhhh sure. Or typo. 😊

Reply:
FIxed in [44df53b](https://github.com/EnigmaCurry/d.rymcg.tech/pull/371/commits/44df53b871c515b617e6fab1e84f753a35ac5a07)